.github/workflows/ci.yml

```diff
@@ -0,0 +1,109 @@
+name: "Build and deploy"
+
+on:
+  push:
+
+jobs:
+  publish:
+    name: Publish to GitHub Pages
+    needs: pdf
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
+
+    steps:
+    - name: Download website build
+      uses: actions/download-artifact@v2
+      with:
+        name: website
+        path: public/
+
+    - name: Download PDF build
+      uses: actions/download-artifact@v2
+      with:
+        name: pdf
+        path: pdf/
+
+    - name: Move PDF files
+      run: mv pdf public/
+
+    - name: Deploy to GitHub Pages
+      uses: peaceiris/actions-gh-pages@v3
+      if: github.ref == 'refs/heads/master'
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        publish_dir: public
+        user_name: 'actions-user'
+        user_email: 'actions@github.com'
+        force_orphan: true
+        cname: guides.etalab.gouv.fr
+
+  pdf:
+    name: Build PDF files
+    needs: build
+    runs-on: ubuntu-latest
+    container: aergus/latex:latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: |
+        apt update
+        apt install -y python3-venv
+
+    - name: Install pandoc-guides
+      run: |
+        git clone https://github.com/AntoineAugusti/pandoc-guides.git /tmp/pandoc-guides
```

Comment:
Will move to etalab org. I did not publish a package on PyPI because it's really tailored to our needs, but we could later have a binary instead.

You can take a look at the code on https://github.com/AntoineAugusti/pandoc-guides if you want to see the logic behind

Reply:
Moved to etalab org ✅